### ISSUE TITLE
Not able to require 'postcss-js' in a comment.js file in my react project

### ISSUE THREAD

**Issue by parthrao** [postcss/postcss-js#3]
I tried to require 'postcss-js' in a component.js file in my react project with webpack compiler which has postcss and postcss-loader, but its showing error that you may need an appropriate loader to handle postcss file type. I have loaded postcss with postcss-loader but still it's not letting me require the module in the .js file . Any suggestions on how to load postcss to make postcss-js work .

**Comment by ai** [comment]
Can you post a error message here? Without it I can't understand what is happen.

Next time write long text to description field :-)

**Comment by parthrao** [comment]
// component.js file
var postcssJs = require('postcss-js');

Getting this error -->
ERROR in ./~/postcss/package.json
Module parse failed: /Users/parth/react-ui-builder-tutorial/relay-starter-kit/node_modules/postcss/package.json Line 2: Unexpected token :
You may need an appropriate loader to handle this file type.
| {
|   "name": "postcss",
|   "version": "5.0.10",
|   "description": "Tool for transforming styles with JS plugins",
 @ ./~/postcss/lib/processor.js 9:15-36

ERROR in ./~/postcss/lib/previous-map.js
Module not found: Error: Cannot resolve module 'fs' in /Users/parth/react-ui-builder-tutorial/relay-starter-kit/node_modules/postcss/lib
 @ ./~/postcss/lib/previous-map.js 19:10-23

**Comment by ai** [comment]
Awesome. Do you have full error stack trace?

**Comment by parthrao** [comment]
This is the full error stack trace.

**Comment by ai** [comment]
Still have no ideas. Browserify and webpack add hooks for `fs` module and should load it.

**Comment by swennemans** [comment]
I got the same error: 

```
ERROR in ./~/postcss/package.json
Module parse failed: /Users/sven/Documents/dev/Elixir/rfmb/node_modules/postcss/package.json Line 2: Unexpected token :
You may need an appropriate loader to handle this file type.
| {
|   "_args": [
|     [
|       "postcss@^5.0.6",
 @ ./~/postcss/lib/processor.js 5:15-36
```

Should I do something in my webpack.config to setup PostCSS?

**Comment by ai** [comment]
@swennemans show me the code, when you load `postcss-js` or `postcss`

**Comment by swennemans** [comment]
My webpack.config

```
module.exports = {
  devtool: prod ? null : 'eval-sourcemaps',
  entry: prod ? entry : [
    'webpack-dev-server/client?' + publicPath,
    'webpack/hot/only-dev-server',
    entry
  ],
  output: {
    path: path.join(__dirname, './priv/static/js'),
    filename: 'bundle.js',
    publicPath: publicPath
  },
  module: {
    loaders: [{
      test: /\.js$/,
      exclude: /node_modules/,
      loaders: loaders
    }, {
      test: /\.css$/,
      loader: 'style!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
    }]
  },
  postcss: [
    require('autoprefixer-core'),
    require('lost')
  ],
  plugins: plugins
};
```

A JS file:
```
import postcssJs from 'postcss-js'

let prefixer = postcssJs.sync([ autoprefixer ]);
let style = prefixer({
    display: 'flex'
});
```

**Comment by ai** [comment]
Do you have any error stack trace? I am not big webpack user and have no idea right now.

**Comment by swennemans** [comment]
I've fixed it. The thrown error was actually pretty good :)

I needed to add the `json-loader` to my webpack config. 
```
  {
      test: /\.json$/,
      loader: 'json-loader'
    },
```

**Comment by ai** [comment]
@swennemans I found a place. I know how to fix it without `json-loader`.

**Comment by ai** [comment]
Fixes in PostCSS 5.0.14.

**Comment by swennemans** [comment]
Awesome! :+1:
